src/webchat-embed/embedded-webchat-styles.css

```diff
@@ -15,6 +15,9 @@
     z-index: 2;
 }
 
+[data-cognigy-webchat-root] [data-cognigy-webchat].webchat .webchat-avatar { 
```

Comment:
This change should be done in `webchat-ui/components/presentational/MessageRow.tsx:57` by changing `flexBasis` to `24` instead of overriding it here.

Reply:
fixed